### PR TITLE
Remove unnecessary instructions for HPA task

### PR DESCRIPTION
**Pull request by Rishit-dagli**
This PR removes unnecessary instructions while demonstrating HorizontalPodAutoscaler.

Fixes: #34317

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Netlify build failed with no particular reason.
@Rishit-dagli please (force-)push again, then a new Netlify build will be triggered.

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md [hunk 1]
```diff
@@ -54,31 +54,8 @@ To learn how to deploy the Metrics Server, see the
...
-    $x += sqrt($x);
-  }
-  echo "OK!";
-?>
-```
-
-Once you have made that container image, start a Deployment that runs a container using the
-image you made, and expose it as a {{< glossary_tooltip term_id="service">}}
+To demonstrate a HorizontalPodAutoscaler, you will first start a Deployment that runs a container using the
+`php-apache` image, and expose it as a {{< glossary_tooltip term_id="service">}}
```
Comment (sftim) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md:58]: This is still wrong I'm afraid. The container image that this manifest runs is: `k8s.gcr.io/hpa-example` (not `php-apache`).

So, 2 things:
- we should change that to `registry.k8s.io/hpa-example` in the manifest
- and we should update the page text to match

Comment (Rishit-dagli) [reply content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md]: Thanks for spotting this!

**Comment by sftim** [comment]
/retitle Remove unnecessary instructions for HPA task

**Comment by sftim** [comment]
I'm happy with this
/lgtm

**Comment by Rishit-dagli** [comment]
/assign @onlydole 
> Once this PR has been reviewed and has the lgtm label, please assign onlydole for approval

**Comment by reylejano** [comment]
Tested the updated manifest that uses `registry.k8s.io/hpa-example` on 1.24.3 and the changes work
/approve

**Pull request merged**